grafana-annotator/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "grafana-annotator.name" -}}
```

Comment:
I've noticed that you've done this in other charts, just wondering what the reasoning is for:
`{{- define "grafana-annotator.name" -}}`
instead of say
`{{- define "name" -}}`

Reply:
from helm docs:
> One popular naming convention is to prefix each defined template with the name of the chart: {{ define "mychart.labels" }}. By using the specific chart name as a prefix we can avoid any conflicts that may arise due to two different charts that implement templates of the same name.